src/main/java/dao/CsvWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package dao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvWriterImpl implements CustomFileWriter {
+    public static final String CSV_DELIMITOR = ",";
+
+    @Override
+    public void writeFile(String filePath, List<String[]> data) {
```

Comment:
the same here. pass full content 
```suggestion
    public void writeFile(String filePath, String content) {
```

Reply:
the same